src/static/browserconfig.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I see this file added but not referenced in site meta - doesn't it need a meta tag with `msapplication-config`?

Reply:
This file is also referenced (commented out) in `config/build-scripts/revision.js` line 49 - do we need to do something with that?

Reply:
Looks like a meta tag is needed
https://docs.microsoft.com/en-gb/previous-versions/windows/internet-explorer/ie-developer/platform-apis/dn320426(v=vs.85)

Reply:
```html
<meta name="msapplication-config" content="IEconfig.xml" />
```

Reply:
will also enable the revision part